compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -9458,9 +9460,14 @@ private void handleSafeNavigation(BLangAccessExpression accessExpr, BType type,
          */
 
         BLangExpression matchExpr = accessExpr.expr;
-        BType matchExprType = accessExpr.expr.getBType();
+        BLangSimpleVariableDef variableDef =
```

Comment:
Shall we update the comment (https://github.com/ballerina-platform/ballerina-lang/pull/38419/files#diff-a5961a28254babde0aa44e4c1cb05126b5ee68494baccca1c76879dec267b8aeR9449-R9459) according to the new desugared code

Reply:
Updated 8959c0339b0f19dbdde5ffbaadc300b8044f9353